docs/guide/head.md

```diff
@@ -20,11 +20,11 @@ order: -1
 
 ## 1. 云服务器
 
-::: info VanBlog
+<!-- ::: info VanBlog
 
 如果不懂什么是云服务器，可以看这篇文章：
 
-:::
+::: -->
```

Comment:
Content is hideen as the link is absent.

Reply:
Because I haven't finished writing that article yet....